python/tests/source.py

```diff
@@ -57,7 +57,7 @@ def dummy_eps(v):
         gv = mp.voltwo(16, 16, 10)
         gv.center_origin()
         sym = mp.mirror(mp.Y, gv)
-        the_structure = mp.structure(gv, dummy_eps, mp._pml(2), sym)
+        the_structure = mp.structure(gv, dummy_eps, mp.pml(2), sym)
```

Comment:
also `mp.PML` here?

Reply:
This test creates low level objects to test typemaps, so I think `mp.pml` is fine.